proposal/proposal.Rmd

```diff
@@ -27,14 +27,17 @@ Depending on project type the detail section should include:
 
 The current version of `volcalc` was created in 2022 as part of a [data science incubator](https://datascience.cct.arizona.edu/cct-data-science-incubator) project in collaboration between Dr. Kristina Riemer and Dr. Laura Meredith at University of Arizona.
 `volcalc` is the first project, to our knowledge, to implement the SIMPOL method for predicting chemical vapor pressures and enthalpies of vaporization [@pankowSIMPOLSimpleGroup2008] in an R package.
-This current version of the `volcalc` package has been successfully used to calculate volatility estimates for *all* 16,000+ compounds in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database (in prep ms ref). validated using hand-collected data [@meredith2022; @honeker2021]
+This current version of the `volcalc` package has been successfully used to calculate volatility estimates for *all* 16,000+ compounds in the KEGG database [@meredith]. 
+To ensure accuracy of the package, the volatility estimates for a subset of the compounds were validated using hand-collected data [@meredith2022; @honeker2021]. 
 
-Because `volcalc` is currently limited to only working with chemical compounds from KEGG, we will refactor it to take molfiles as inputs directly.
+Because `volcalc` is currently limited to only working with chemical compounds from KEGG, we will refactor it to take chemical compound structure files as inputs directly.
+The type of structure file the packagee uses is called a molfile. 
 The main function in `volcalc`, `calc_vol()`, currently downloads chemical information as molfiles from the KEGG API as a starting point.
-It reads in that .mol file and parses it to find functional groups.
+It reads in that molfile and parses it to find functional groups.
 It finally applies an algorithm (published in @pankowSIMPOLSimpleGroup2008) to predict vapor pressure, and outputs to the user a relative measure of volatility.
 
-Molfile is an open format and various tools exist to translate other standard representations of chemical structure such as SMILES [@weininger1988smiles] and InChI [@heller2013] to molfiles (e.g. using the [`OpenBabel`](http://openbabel.org/wiki/Main_Page) command line utility or in R with [`ChemmineOB`](https://github.com/girke-lab/ChemmineOB)).
+A molfile is an open format, and various tools exist to translate other standard representations of chemical structure such as SMILES [@weininger1988smiles] and InChI [@heller2013] to molfiles. 
```

Comment:
I think it was correct before.  Like saying "CSV is an open format" or "XLSX is a proprietary format".
```suggestion
Molfile is an open file format, and various tools exist to translate other standard representations of chemical structure such as SMILES [@weininger1988smiles] and InChI [@heller2013] to molfiles. 
```